test/e2e_node/memory_eviction_test.go

```diff
@@ -101,8 +101,30 @@ var _ = framework.KubeDescribe("MemoryEviction [Slow] [Serial] [Disruptive]", fu
 
 			}, 60*time.Minute, 5*time.Second).Should(Equal(true))
 
+			// Wait for the memory pressure condition to disappear from the node status before continuting.
+			Eventually(func() bool {
+				nodeList, err := f.Client.Nodes().List(api.ListOptions{})
+				if err != nil {
+					glog.Errorf("Tried to get node list but got error: %v", err)
+					return false
+				}
+				// Assuming that there is only one node, because this is a node e2e test.
+				if len(nodeList.Items) != 1 {
+					glog.Errorf("Expected 1 node, but see %d.", len(nodeList.Items))
+					return false
+				}
+				node := nodeList.Items[0]
+				for _, c := range node.Status.Conditions {
+					if c.Type == api.NodeMemoryPressure {
```

Comment:
Why not the check the condition status? The presence of the condition alone is not an issue. if its status is `true` that's when we have an issue.